README.md

```diff
@@ -4,6 +4,9 @@ Wrapping Google tagmanager and adding some helpers to him
 * This script appends automatically the googleTagmanager script.
 * It also adds an iframe inside a noscript tag for non-js fallback
 
+### [Documentation](https://getninjas.github.io/tagmanager-wrap/)
```

Comment:
This could be:

```Markdown
### Documentation

See live here [https://getninjas.github.io/tagmanager-wrap](https://getninjas.github.io/tagmanager-wrap)
```

Reply:
@eduardojmatos Nice! 🥇 Will do it.